org.eclipse.xtext.xbase.ui/.classpath

```diff
@@ -4,6 +4,13 @@
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins">
 		<accessrules>
 			<accessrule kind="accessible" pattern="org/eclipse/jdt/internal/ui/actions/ActionMessages**"/>
+			<accessrule kind="accessible" pattern="org/eclipse/jdt/internal/core/DefaultWorkingCopyOwner"/>
```

Comment:
am not sure if i want this. this hides internal usage warning. correct?

Reply:
Correct, right now only warnings for ActionMessages** were ignored. I can understand not wanting to hide those. However, the warnings are only useful if people actually look at them, not sure if that is the case.

Reply:
we plan
(1) to reduce internal api
(2) become xfriends with the eclipse internal api we use so that we get notified on changes.
this is why i want to keep the warnings for now

Reply:
Sounds like a good idea to me, I'll revert this change.